automated-comments/ruby/two-fer/string_format.md

```diff
@@ -1 +1,9 @@
-As well as the `String#%%` method, a common way to create strings in Ruby is to use interpolation. For example, rather than `"One for %%s, one for me." %% %<name_variable>s` you might consider `"One for #{%<name_variable>s}, one for me."`.
+As well as the `String#%%` method, a common way to create strings in Ruby is to use [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
+
+```ruby
```

Comment:
Isn't giving the solution going against the suggested automated-mentoring guidelines: https://github.com/exercism/automated-mentoring-support/pull/61/files#diff-c3fc7e01c1c858a88579c56982ac5ba5R60?

Reply:
> However, if the discovery **doesn't** trigger a dopamine hit, it makes total sense to show what it looks like.

I think that's the case here.